### ISSUE TITLE
JetEngine - Custom taxonomy not visible with shop_manager role

### ISSUE THREAD

**Issue by rtpHarry** [CrocoBlock/suggestions#6299]
**Describe the bug**
Something strange seems to be going on with the capabilities, JetEngine and the WooCommerce shop_manager role.

If you look at the default capability type of a JetEngine custom taxonomy it lists it as `post`:

<img width="567" alt="image" src="https://user-images.githubusercontent.com/1038062/209011232-f34e0e8c-2c0c-4b41-82b3-4b0f0a32072e.png">

If you look at the default capability type of a JetEngine custom type it lists it as `post`:

<img width="576" alt="image" src="https://user-images.githubusercontent.com/1038062/209011159-17ff6d5d-aecb-42c9-b092-24d8efe4eb03.png">

I can't find 'post' as a cap anywhere in WP or WooCommerce. There is `edit_posts`, which is assigned to `shop_manager`:

https://github.com/woocommerce/woocommerce/blob/ee01d4219282387c2975ef4594677453c1dd7a0e/includes/class-wc-install.php#L1052

and `administrator`:

https://github.com/WordPress/wordpress-develop/blob/e50c65caf4999b11eb396ceaf6649b9818fab47f/src/wp-admin/includes/schema.php

For some reason though, it works for normal Admin users. I can see custom post types and the custom taxonomy.

The `shop_manager` can see the custom post type, but the taxonomy is missing.

It doesn't seem to matter what I change the taxonomy capability to. Even if I set it to jibberish like below then it still displays that taxonomy for the `administrator` role, but not for the `shop_manager` role:

<img width="767" alt="image" src="https://user-images.githubusercontent.com/1038062/209010000-4074b21a-68c7-44c8-af7a-05bc08f57f0f.png">

Look at the code, it seems that it is turning it into `manage_post` which I can't find as a string in the WP or WC source code, but it might be generated dynamically:

<img width="555" alt="image" src="https://user-images.githubusercontent.com/1038062/209009849-5283a712-f76f-4376-9a26-7f00a199a4cb.png">

I also tried setting it to `term` and `terms` which didn't work for `shop_manager`.

**If I clear out the capability type to an empty string then it does work for `shop_manager`.**

(I would be grateful if anyone reading this could answer if I've created a security hole by doing that though)

So to summarise; something is strange here. 

  - The `shop_manager` is a high-level account that can edit posts and pages, so it should be able to edit the custom taxonomy. 
  - The capability type of a custom taxonomy seems to have no effect on the `administrator` role (it always shows). 
  - And the only way for it to show for `shop_manager` is to remove the default `post` string and save it with a blank string

**Comment by Floydtm** [comment]
You can install the role editor plugin and configure the shop_manager role as you need. Let me know if this helps you
